x-pack/legacy/plugins/actions/server/action_type_registry.ts

```diff
@@ -10,28 +10,35 @@ import { ActionType, GetServicesFunction } from './types';
 import { TaskManager, TaskRunCreatorFunction } from '../../task_manager';
 import { getCreateTaskRunnerFunction, ExecutorError } from './lib';
 import { EncryptedSavedObjectsPlugin } from '../../encrypted_saved_objects';
+import { SpacesPlugin } from '../../spaces';
 
 interface ConstructorOptions {
   taskManager: TaskManager;
   getServices: GetServicesFunction;
   encryptedSavedObjectsPlugin: EncryptedSavedObjectsPlugin;
+  spaceIdToNamespace: SpacesPlugin['spaceIdToNamespace'];
```

Comment:
It seems to be the way TypeScript likes to access sub-types of a type. Otherwise I get errors like `Cannot access 'SpacesPlugin.spaceIdToNamespace' because 'SpacesPlugin' is a type, but not a namespace.`